app/assets/stylesheets/pregnancies.scss

```diff
@@ -59,3 +62,92 @@
     }
   }
 }
+
+.modal.pledge_modal {
+	display: none;
+  height: 475px;
+  width: 500px;
+  position: fixed;
+  margin: 0 auto;
+  left: calc(50% - 250px);
```

Comment:
Here's how this renders for me: 

<img width="1211" alt="screen shot 2016-05-28 at 3 41 01 pm" src="https://cloud.githubusercontent.com/assets/3866868/15629586/ba3de512-24ea-11e6-8b38-04ba7bfa16bb.png">

I defer to you and @rebeccaestes @mebates but I suspect we'd want this centered, no? Just want to confirm that this is the proper behavior!